src/index.css

```diff
@@ -0,0 +1,18 @@
+@import url("@fontsource/open-sans");
+
+:root {
+  font-family: "Open Sans", sans-serif;
```

Comment:
Yeah, I think I prefer this. The argument for using a "system" font stack is that is looks like it "belongs" in the OS it's running in - especially important for things that users think of as "apps". It also makes for a smaller download. But, this way gets us consistency at the cost of ~14K of web fonts.